src/main/java/codeu/model/data/Message.java

```diff
@@ -38,8 +41,11 @@
   public Message(UUID id, UUID conversation, UUID author, String content, Instant creation) {
     this.id = id;
     this.conversation = conversation;
+    _processedContent = content;
```

Comment:
Again to follow the style of this file, I would change this to

this.processedContent = content;